files/build_templates/docker_image_ctl.j2

```diff
@@ -27,6 +27,21 @@ link_namespace() {
 }
 {%- endif %}
 
+function updateSyslogConf()
+{
+    # On multiNPU platforms, change the syslog target ip to docker0 ip to allow logs from containers
+    # running on the namespace to reach the rsyslog service running on the host
+    # Also update the container name  
+    if [[ ($NUM_ASIC -gt 1) ]]; then
+        TARGET_IP=$(ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1)
+        CONTAINER_NAME="{{docker_container_name}}$DEV"
+        TMP_FILE="/tmp/rsyslog.$CONTAINER_NAME.conf"
+        sonic-cfggen  -t /usr/share/sonic/templates/rsyslog-container.conf.j2 -a "{\"target_ip\": \"$TARGET_IP\", \"container_name\": \"$CONTAINER_NAME\" }"  > $TMP_FILE
```

Comment:
>  [](start = 21, length = 1)

One extra blank

Reply:
added in the latest commit

Reply:
Could you remove the extra blank in `sonic-cfggen  -t`

---
In reply to: [439579965](https://github.com/Azure/sonic-buildimage/pull/4738#discussion_r439579965) [](ancestors = 439579965)

Reply:
Fixed in the latest commit